back-end/routes/api/users.js

```diff
@@ -39,6 +39,7 @@ router.post("/register", async (req, res) => {
   // There was not an existing user, create a new one
   const newUser = new User({
     name: req.body.name,
+    username: req.body.username,
```

Comment:
before doing this, you will want to check if the username is already in use (similar to checking if the email is already in use)